constraints.txt

```diff
@@ -92,7 +92,7 @@ pytest==4.1.1
 python-dateutil==2.7.5
 pytoml==0.1.19
 pytz==2018.5
-raiden-contracts==0.10.1
+raiden-contracts==0.11.0
```

Comment:
Just to make sure this is intended: This will change from limitless contracts to the contracts with limits and constraints enabled.
I'm a bit surprised when tests would pass, because that would mean that we never test multiple token networks.